required/latex-lab/latex-lab-math.dtx

```diff
@@ -114,11 +115,142 @@
+% |\m@th| sets also |\mathsurround| to zero, which is often
+% wanted in untagged math anyway.
+%
+% Text tagging is \emph{not} suppressed inside the math, e.g.,
+% |\mbox{\emph{text}}| will still create an Em-structure.
+% In contrast nested math is not tagged.
+%
+% To suppress the text tagging |\SuspendTagging{}| can be used:
+%    \begin{itemize}
+%    \item no math tagging, but |\emph| is tagged:\\
+%          |$\m@th\langle\mbox{\emph{if and only if} $x=y$}\rangle$|
+%
+%    \item no internal tagging:
+%          \\
+%         |$\m@th\SuspendTagging{}\langle\mbox{\emph{if and only if}} $x=y$}\rangle$|
+%    \end{itemize}
+%
+% The method is quite suitable for symbols and also for sub- and superscripts
+% (as long as they don't contain nested math that should be tagged).
+%
+%
+% \subsubsection{Using \cs{m@th} \emph{before} the opening \texttt{\$}}
+%
+% |\m@th| (as defined in the latex-lab-math code) sets also the internal boolean
+% which controls the grabbing  to true and so when it is used \emph{before}
+% some math it disables math grabbing and tagging for all following math
+% in the current group (including nested math).
+% Text tagging inside the math is still active, it can be disabled as above
+% with |\SuspendTagging{}|.
+%
+% When |\m@th| is used like this it is possible to reenable the math
+% tagging of nested math, by adding |\MathCollectTrue| after the opening |$|.
+%
+%    \begin{itemize}
+%    \item no math tagging, but |\emph| is tagged:\\
+%        |{\m@th$\langle\mbox{\emph{if and only if} $x=y$}\rangle$}|
+%
+%    \item  both nested math and |\emph| is tagged:\\
+%        |{\m@th$\MathCollectTrue\langle\mbox{\emph{if and only if} $x=y$}\rangle$}|
+%    \end{itemize}
+%
+% The method is suitable for symbols and sub- and superscripts too
+% (it is actually how superscripts avoid math tagging currently).
+% It can also be used in cases where the end math shift token is
+% hidden in some other command. But it is not so useful for
+% larger boxes as it sets |\mathsurround| to zero.
+% Grouping should be used to avoid side-effects on following math.
+%
+%
+%\subsubsection{Disabling math tagging with \cs{MathCollectFalse}}
+%
+% Without a side-effect on |\mathsurround| the math grabbing can be
+% suppressed and reenabled by setting the boolean that controls the collecting.
+% So in the following example the math in the |\mbox| is properly tagged,
+% but the angled brackets are simple text.
+%
+%    \begin{verbatim}
+%    %stop math grabbing
+%    \MathCollectFalse
+%    $
+%    %(optional) restart math grabbing for nested math
+%    \MathCollectTrue
+%    \langle\mbox{\emph{if and only if}} $x=y$}\rangle
+%    $
+%    %(optional) restart math grabbing for following math
+%    % if there is no grouping
+%    \MathCollectTrue
+%    \quad $x=1$
+%    \end{verbatim}
+%    The method is suitable if a box should be centered with
+%    |\vcenter| (and is used in array.sty for the tabular code).
```

Comment:
is it? Perhaps one has to rethink all of this. After all this all gets you \mathsurround while in reality (if it is positiv) you wouldn't really want that around \vcenter or a single \langle.

so maybe it is after all a combination of \m@th plus \MathCollect...